frontend/src/types/NMPFileFarmManureData.ts

```diff
@@ -1,4 +1,4 @@
-import { ManureNutrients } from './calculateNutrients';
+import { ManureNutrients } from './CalculateNutrients';
```

Comment:
Did you rename the file?

Reply:
No, I branched off someone else who did. But it appears they did not update the imports. Going through and updating them.